BlazarService/src/main/java/com/hubspot/blazar/util/SlackUtils.java

```diff
@@ -148,4 +153,10 @@ private static SlackAttachment makeAttachment(String title) {
     return new SlackAttachment(title, title, "", "");
   }
 
+  public static Retryer<Boolean> makeSlackMessageSendingRetryer() {
+    return RetryerBuilder.<Boolean>newBuilder()
+        .retryIfResult(Predicates.equalTo(Boolean.FALSE))
+        .withWaitStrategy(WaitStrategies.exponentialWait(3000, TimeUnit.MILLISECONDS))
```

Comment:
I believe we just had someone discover this wait strategy doesn't work as expected